projects/components/src/popover/popover.component.ts

```diff
@@ -90,4 +121,29 @@ export class PopoverComponent implements OnDestroy {
 
     this.popoverOpen.emit(this.popover);
   }
+
+  private handleHoverPopoverTrigger(event: 'mouseenter' | 'mouseleave'): void {
+    if (this.disabled) {
+      return;
+    }
+
+    switch (event) {
+      case 'mouseenter':
+        this.popoverHoverTriggerService.showPopover({
```

Comment:
Clicking on a trigger and hovering a trigger shouldn't go down different code paths. At least the bulk should be shared, could have different listeners etc.

Reply:
In fact, it looks like this path is just doing its own thing and ignoring all the inputs and outputs.